offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/create/CreateOfferViewModel.groovy

```diff
@@ -234,4 +272,17 @@ class CreateOfferViewModel {
             }
         }
     }
+
+    /**
+     * Sets the customer for the offer. Whenever a change is detected an event is emitted.
+     * @param customer
+     */
+    void setCustomer(Customer customer) {
+        if (this.customer != customer) {
+            personUpdateEvent.emit(customer)
+        } else {
+            log.warn("Overwrite of equal values. ${this.customer} to be overwritten with $customer.")
+        }
```

Comment:
Why this warning? Nothing bad should happen here, right?

Reply:
I would rather check for null

Reply:
I do check for null, removing a selection should be okay as well. So null is not a case where you would expect something bad to happen here.

Reply:
In this case I think the notifcation is not required

Reply:
```suggestion
        } 
```

Reply:
I think the notification still makes sense since we do not fire an event in the case of two equal values. Maybe the scope needs adjustment to debug though :+1: